manpage/kosmorro.1.md

```diff
@@ -26,7 +26,7 @@
     the timezone to display the hours in; e.g. 2 for UTC+2 or -3 for UTC-3
 
 `--no-colors`  
-    disable the colors in the console
+    disable the colors in the console. Set an environment variable `NO_COLORS` to permanently disable colored console output.
```

Comment:
This should be documented in the _Environment variables_ section below.

Reply:
Oops! I missed that somehow. I moved this bit to that section.